pkg/gardenlet/controller/seed/seed/reconciler_reconcile.go

```diff
@@ -377,6 +377,17 @@ func (r *Reconciler) runReconcileSeedFlow(
 	// Deploy the CRDs in the seed cluster.
 	log.Info("Deploying custom resource definitions")
 
+	if !seedIsGarden {
```

Comment:
Please adapt https://github.com/gardener/gardener/blob/master/test/integration/gardenlet/seed/seed/seed_test.go to cover this case